### DISCUSSION TITLE
PJ-1203A Meter reset

### DISCUSSION THREAD

**Discussion by Minims** [Koenkk/zigbee2mqtt#23544] [Devices]
Hello, 

is there a way to reset PJ-1203A energy meter like the zigbee plug ? 
https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html#reset-energy 

Thanks

**Comment by ppluciennik** [comment]
You must press physical button 5 times (quite fast).

**Reply by Minims** [reply]
Thanks for reply. So it needs to be accessible
